member/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Member(models.Model):
+    user = models.OneToOneField(User)
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+
+    def __str__(self):
+        return self.user.get_username()
+
+    def __unicode__(self):
+        return self.user.get_username()
```

Comment:
I'm wondering whether ``python-social-auth`` provides it's way to store data in ``django.contrib.auth.models.User``
Or maybe your implementation enables a user to login with two ways?